internal/core/item_tree.rs

```diff
@@ -292,12 +292,18 @@ impl ItemRc {
                 // Loop: We stepped into an empty repeater!
             } else {
                 // Step out of this component:
-                let root = ItemRc::new(self.component(), 0);
+                let mut root = ItemRc::new(self.component(), 0);
                 if let Some(item) = subtree_step(root.clone()) {
                     return step_in(item);
                 } else {
                     // Go up a level!
-                    if let Some(parent) = root.parent_item().upgrade() {
+
+                    let mut parent_item = Default::default();
```

Comment:
wouldn't `parent_node` be a better name?

Reply:
It indeed is.